package.json

```diff
@@ -50,6 +51,76 @@
     "chai": "^3.5.0",
     "vscode-textmate": "^2.1.1"
   },
+  "runtimeDependencies": [
```

Comment:
OK. This should work now. You should be able to specify an array of `runtimeIds` for a particular runtime dependency in the package.json file and the PackageManager will download and install it when the current RID matches. A couple of caveats:

1. You package must be a .zip file
2. The .zip file will be extracted in the extension root path. So, any directory structure must be included in the .zip.
3. `gulpfile.js` will need to be updated once the debugger moves its acquisition to the package manager.
4. We need to add telemetry to the package manager.